spec/services/idv/steps/in_person/state_id_step_spec.rb

```diff
@@ -180,7 +168,11 @@
 
           # On Verify, user changes response from "No,..." to
           # "Yes, I live at the address on my state-issued ID
-          step.call
+          @result = step.call
+        end
+
+        it 'retains identity_doc_ attrs/value ands addr attr
+        with same value as identity_doc in flow session' do
```

Comment:
I think it's bad to have a string literal broken over 2 lines, it's more common as a heredoc
```suggestion
        it <<~EOS.squish do
          retains identity_doc_ attrs/value ands addr attr
          with same value as identity_doc in flow session
        EOS
```

Reply:
There a few instances of spec descriptions broken over 2 lines in this file.  I would be grateful if you were able to clean them up if you don't mind while in this file.  🙏